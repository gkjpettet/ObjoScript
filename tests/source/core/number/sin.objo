System.print(0.sin) # Expect 0
System.print(1.sin) # Expect 0.841470985
System.print(0.5.sin) # Expect 0.479425539