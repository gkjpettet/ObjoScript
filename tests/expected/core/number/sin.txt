0
0.841470985
0.479425539